manifests/charts/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -117,6 +117,12 @@ rules:
 {{- end }}
     verbs: ["approve"]
 {{- end}}
+{{- if (.Values.env).ENABLE_CLUSTER_TRUST_BUNDLE_API }}
+  - apiGroups: ["certificates.k8s.io"]
+    resources:
+      - "clustertrustbundles"
+    verbs: ["update", "create", "delete"]
```

Comment:
can we scope to a single name (istio-ca-root-cert)? I _think_ it should work but the per-name permissions are a little wonky in k8s so not certain

Reply:
will try it

Reply:
turns out just using `resourceName` does not work, but it works in conjunction with `signerName`. I am using that now